### PR TITLE
Added Markdown Cheat sheet

### PR DESCRIPTION
**Pull request by oyenuga17**
- #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,4 +27,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
 
 <!-- ALL-CONTRIBUTORS-LIST:END -->
 
-This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind welcome!
\ No newline at end of file
+This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind welcome!
+
+## Github markdown resources
```
Comment (Ifycode) [inline README.md:32]: Neatly written and done! Well done 👍
But your new section is in the wrong place. Check the issue again to see where you should have added it.

Comment (oyenuga17) [reply README.md]: I have fixed it,

**Comment by Ifycode** [comment]
And then you low-key fixed it @oyenuga17 😂👍
Nicely done. Approved 💯😘

Once I merge, pull the new content from my main branch to your own main branch as usual.

File: README.md (hunk 1 above)
Comment (Ifycode) [reply README.md]: 💯🙂👍

**Comment by oyenuga17** [comment]
Okay..I will do that.

**Pull request merged**